### PR TITLE
Add release notes, supported version details and update download links for 5.21.1 release

### PR DESCRIPTION
**Pull request by cwjohnston**
## Description

* Update release notes to include 5.21.1 fixes and improvements
* Update version and build numbers on the install and platform pages
* Add 5.21.1 to the supported versions page
* Update `update-sensu-version.sh` to account for changes to some document paths

## Motivation and Context

Changes herein are required to ship the 5.21.1 patch release.

## Review Instructions

Please :eyes: my adhoc release synopsis for clarity, etc. I'll copy changes over to 6.0 once review is approved.

### CODE REVIEW

File: content/sensu-go/5.21/release-notes.md [hunk 1]
```diff
@@ -65,6 +66,23 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.21.1 release notes
+
+**August 5, 2020** &mdash; The latest release of Sensu Go, version 5.21.1, is now available for download.
+
+This patch release includes fixes for a Web UI crash when interacting with namespaces containing a thousand or more events, regressions in logging of various agent errors, and an enhancement which provides additional context to websocket connection errors logged by the backend.
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/release-notes.md]:
```suggestion
This patch release includes fixes for a web UI crash when interacting with namespaces that contain 1000 or more events and regressions in logging various agent errors as well as an enhancement that provides additional context to websocket connection errors logged by the backend.
```

File: content/sensu-go/5.21/release-notes.md [hunk 2]
```diff
@@ -65,6 +66,23 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.21.1.
+
+**IMPROVEMENTS:**
+
+- Backend log messages related to connection errors on the Agent Websocket API now provide more context about the error.
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/release-notes.md]:
```suggestion
- Backend log messages related to connection errors on the agent websocket API now provide more context about the error.
```

File: content/sensu-go/5.21/release-notes.md [hunk 3]
```diff
@@ -65,6 +66,23 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**FIXES:**
+
+- Fixed a potential Web UI crash when fetching events in namespace with > 1000 events.
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/release-notes.md]:
```suggestion
- Fixed a potential web UI crash when fetching events in namespace with 1000 or more events.
```

File: content/sensu-go/5.21/release-notes.md [hunk 4]
```diff
@@ -65,6 +66,23 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- Fixed a regression which prevented errors produced in the agent by assets, check validation, token substitution, and event unmarshaling from being logged.
```
Comment (hillaryfraley) [inline content/sensu-go/5.21/release-notes.md]:
```suggestion
- Fixed a regression that prevented errors produced in the agent by assets, check validation, token substitution, or event unmarshaling from being logged.
```

**Review by hillaryfraley — APPROVED** [review]
Looks great! Everything works for me locally 👍

**Comment by calebhailey** [comment]
Oh no... my ~workaround~ script has migrated to another repo! haha 😅

**Pull request merged**